Index: Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>public class Main {\n    public static void Main(String args[]) {\n\n    }\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Main.java b/Main.java
--- a/Main.java	(revision 6d3a33a96fe5596438b9c050c961e7e43dfecaec)
+++ b/Main.java	(date 1650885500522)
@@ -1,5 +1,5 @@
 public class Main {
     public static void Main(String args[]) {
-
+        System.out.println("Hello world!");
     }
 }
